scripts/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+password_url="taskcluster/secrets/v1/secret/repo:github.com/mozilla/balrog:dockerhub"
```

Comment:
This will need to be :coveralls instead of :dockerhub. I'm working on getting the secret added still, but it will be there soon!